openehr-rm/src/main/java/com/nedap/archie/rminfo/UpdatedValueHandler.java

```diff
@@ -35,6 +40,44 @@
         if(parent instanceof CodePhrase) {
             return fixCodePhrase(rmObject, archetype, pathOfParent);
         }
+        if (parent instanceof DvQuantity) {
+            return fixDvQuantity(rmObject, archetype, pathOfParent);
+        }
+
+        return new HashMap<>();
+    }
+
+    private static Map<String, Object> fixDvQuantity(Object rmObject, Archetype archetype, String pathOfParent) {
+        try {
+            Map<String, Object> result = new HashMap<>();
```

Comment:
Two more todo's:
- check if actually the magnitude is being set (see the pathOfParent.endsWith contruction elsewhere)
- if units or precision already have a value, should they be set? You probably do not want to override a value set manually or with a different rule elsewhere?

Reply:
>check if actually the magnitude is being set (see the pathOfParent.endsWith contruction elsewhere)

I only see `.../value` in the `pathOfParent` when these quantities are being set by rule evaluation. Not sure how that would work then, or should I expect a `.../value/magnitude` path or something.